src/main/java/org/triplea/ui/TripleaMainMenu.java

```diff
@@ -0,0 +1,174 @@
+package org.triplea.ui;
+
+import java.awt.Desktop;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.text.MessageFormat;
+
+import games.strategy.debug.ClientLogger;
+import games.strategy.engine.ClientContext;
+import games.strategy.triplea.UrlConstants;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.Button;
+import javafx.scene.control.PasswordField;
+import javafx.scene.control.TextField;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Text;
+
+public class TripleaMainMenu extends BorderPane {
```

Comment:
They don't need any prefix or special class name, also I believe they don't need to be public.
(Although it is possible to require a specific class name via the FXML file to avoid calling `setController(this)`)